<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second_window</class>
 <widget class="QWidget" name="second_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(227, 183, 183);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="Title">
      <property name="styleSheet">
       <string notr="true">color: rgb(224, 27, 36);</string>
      </property>
      <property name="text">
       <string>CRIMSON TECH</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="production_unit_2">
      <property name="styleSheet">
       <string notr="true">color:rgb(224, 27, 36)</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Procuction Unit &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>691</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>60</y>
     <width>20</width>
     <height>521</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Home_icon">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Weight_counter_first/fugue-icons-3.5.6/icons/application-home.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>106</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Packet Sachet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Weight_counter_first/fugue-icons-3.5.6/icons/arrow-curve-000-left.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Non-veg</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>601</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>Wai Wai chicken </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton_10">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton_11">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton6">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>9</width>
        <height>4</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton5">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton9">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton8">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton4">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton7">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton_12">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>wai wai</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton3">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { color: red; background-color: rgb(143, 240, 164) }
QPushButton:enabled { color: red }
QPushButton:hover { color: blue }</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeWidget::branch:has-siblings:!adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:has-siblings:adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:!has-children:!has-siblings:adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:closed:has-children:has-siblings {
    background-color: lightgray;
}

QTreeWidget::branch:open:has-children:has-siblings {
    background-color: lightgray;
}

QHeaderView::section {
    background-color: lightblue;
    border: 1px solid black;
    padding: 2px;
}
</string>
    </property>
    <column>
     <property name="text">
      <string>Cake</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Veg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Non-veg</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeWidget::branch:has-siblings:!adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:has-siblings:adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:!has-children:!has-siblings:adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:closed:has-children:has-siblings {
    background-color: lightgray;
}

QTreeWidget::branch:open:has-children:has-siblings {
    background-color: lightgray;
}

QHeaderView::section {
    background-color: lightblue;
    border: 1px solid black;
    padding: 2px;
}
</string>
    </property>
    <column>
     <property name="text">
      <string>Packet-Sachet</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Veg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Non-veg</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeWidget::branch:has-siblings:!adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:has-siblings:adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:!has-children:!has-siblings:adjoins-item {
    background-color: lightgray;
}

QTreeWidget::branch:closed:has-children:has-siblings {
    background-color: lightgray;
}

QTreeWidget::branch:open:has-children:has-siblings {
    background-color: lightgray;
}

QHeaderView::section {
    background-color: lightblue;
    border: 1px solid black;
    padding: 2px;
}
</string>
    </property>
    <column>
     <property name="text">
      <string>Twin Sachet</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Veg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Non-veg</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
